packages/next-dashboard/src/utils/configContext.js

```diff
@@ -0,0 +1,68 @@
+// @flow
+import React from 'react';
+
+// Types used in comment-notation syntax
+// Some editors freak out if not using comment-notation for typecasting.
+// eslint-disable-next-line no-unused-vars
+import type { Branding, Theme, IAuthProvider } from './types';
+import type { NextComponent } from './nextTypes';
```

Comment:
Hmm, couldn't we use `/*:: */` comment notation with Flow around the imports rather than muting the unused-vars warning from eslint? It feels slightly less dirty, maybe... (the comment about why comment notation is used is good and should be kept)